ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -651,6 +663,10 @@ mac_address_table:
           source_address: < ipv4_address -> default ip address of Loopback0 >
           version: < 1, 2, 3 -> default 2 (EOS) >
 
+        # Explicitly extend this VLAN to remote EVPN domains.
+        # Overrides tenants.<tenant>.evpn_multi_domain ( default --> true ).
+        evpn_multi_domain: < true | false | default --> true >
```

Comment:
This is not default true. This inherits from tenant, so it has no default

Reply:
Resolved!